source/common/ssl/context_config_impl.cc

```diff
@@ -25,10 +25,13 @@ const std::string ContextConfigImpl::DEFAULT_CIPHER_SUITES =
     "AES256-SHA256:"
     "AES256-SHA";
 
+const std::string ContextConfigImpl::DEFAULT_ECDH_CURVES = "X25519:P-256";
```

Comment:
@mattklein123 my question was mostly to check if changing the default curve was acceptable. Those values are sane (Google's public facing infrastructure is using those curves as well).

@davidben would you mind commenting on the curves and/or looking at this PR? Thanks!